src/nectarchain/calibration/NectarGain/PhotoStat/PhotoStat.py

```diff
@@ -0,0 +1,268 @@
+import numpy as np
+from scipy.stats import linregress
+from matplotlib import pyplot as plt
+import astropy.units as u
+from astropy.visualization import quantity_support, time_support
```

Comment:
`time_support` is not used, remove import ?

Reply:
yes I forgot to remove